packages/bff/src/services/catalogService.ts

```diff
@@ -537,5 +544,53 @@ export function catalogServiceBuilder(
         file: Buffer.from(buildCsv(consumers)),
       };
     },
+    exportEServiceDescriptor: async (
+      eserviceId: EServiceId,
+      descriptorId: DescriptorId,
+      context: WithLogger<BffAppContext>
+    ): Promise<void> => {
+      const requesterId = context.authData.organizationId;
+      const headers = context.headers;
+
+      const eservice = await catalogProcessClient.getEServiceById({
+        params: {
+          eServiceId: eserviceId,
+        },
+        headers,
+      });
+
+      assertRequesterIsProducer(requesterId, eservice);
+
+      const descriptor = retrieveEserviceDescriptor(eservice, descriptorId);
+
+      if (!descriptor.interface) {
+        throw missingInterface(eserviceId, descriptorId);
+      }
+    },
```

Comment:
this function is used?

Reply:
something was wrong this method should be part only of this [PR](https://github.com/pagopa/interop-be-monorepo/pull/882)

Reply:
This is a leftover, removed in this [commit](https://github.com/pagopa/interop-be-monorepo/pull/838/commits/25aeace426ad5c2ebc44c57979c4bae862065cdc) 🧹